apis/nucleus/src/components/listbox/ListBoxInline.jsx

```diff
@@ -56,13 +56,14 @@ const StyledGrid = styled(Grid, { shouldForwardProp: (p) => !['containerPadding'
   })
 );
 
-const Title = styled(Typography)(({ theme }) => ({
+const Title = styled(Typography)(({ theme, isPopover }) => ({
   color: theme.listBox?.title?.main?.color,
   fontSize: theme.listBox?.title?.main?.fontSize,
   fontFamily: theme.listBox?.title?.main?.fontFamily,
   fontWeight: theme.listBox?.title?.main?.fontWeight || 'bold',
+  textOverflow: isPopover ? 'ellipsis' : undefined,
+  overflow: isPopover ? 'hidden' : undefined,
```

Comment:
why only for isPopover?

The not popover case can also have overflow:
If it work then without extra styling why is it needed for popover, 
or maybe it don't work and the styling should always be added.

Reply:
yeah, seems the overflow style already exists, so agreed that we do not need specific case for popover. Shall remove it.